fiftyone/utils/labelstudio.py

```diff
@@ -575,11 +578,12 @@ def generate_labeling_config(media, label_type, labels=None):
     return config_str
 
 
-def import_label_studio_annotation(result):
+def import_label_studio_annotation(result, label_type):
```

Comment:
Can you make `label_type=None` the default, since this extra information is only needed when deciding between semantic and instance segmentations?